### ISSUE TITLE
Does Best Buy care about running a bot?

### ISSUE THREAD

**Issue by JayV30** [nickconnors/RTX-3070-Best-Buy-Bot#2]
Hi Nick - this is a nice little bot! I'm looking to get a current-gen video card and was about to write a bot using the Best Buy API but I thought I'd do a quick search of GitHub first to save myself some time.

I really like what you've done here, automating the checkout process instead of just sending an alert (which is what I was going to do).

I have a few quick questions if you have time (if not, don't worry about it and just close this issue - no worries).

1. Did this work for you? Did you get a card using your bot?
2. In your experience using this, did Best Buy take any action to prevent automations and crawlers? I'm sitting here watching the bot run and wondering if they are going to take any preventative measures like Amazon does.

Thank again for creating this - certainly has saved me a bit of time!

**Comment by nickconnors** [comment]
Hey, thanks for taking a look at my bot. First off, yes, I was able to successfully get the card. To answer your next question, at the time Best Buy did not do anything about the bots. However, I believe they have since implemented some sort of 2 factor system to combat the bots. I don't know much other than that because I only used this bot on launch day. I'm not interested in scalping, just wanted the card for personal use

**Comment by nickconnors** [comment]
> Hi Nick - this is a nice little bot! I'm looking to get a current-gen video card and was about to write a bot using the Best Buy API but I thought I'd do a quick search of GitHub first to save myself some time.
> 
> I really like what you've done here, automating the checkout process instead of just sending an alert (which is what I was going to do).
> 
> I have a few quick questions if you have time (if not, don't worry about it and just close this issue - no worries).
> 
> 1. Did this work for you? Did you get a card using your bot?
> 2. In your experience using this, did Best Buy take any action to prevent automations and crawlers? I'm sitting here watching the bot run and wondering if they are going to take any preventative measures like Amazon does.
> 
> Thank again for creating this - certainly has saved me a bit of time!

Hey, thanks for taking a look at my bot. First off, yes, I was able to successfully get the card. To answer your next question, at the time Best Buy did not do anything about the bots. However, I believe they have since implemented some sort of 2 factor system to combat the bots. I don't know much other than that because I only used this bot on launch day. I'm not interested in scalping, just wanted the card for personal use

**Comment by JayV30** [comment]
Thanks for writing back! I just came back to my home office and found the script still running. Obviously I have no idea when stocks will be released, but hopefully I will be working and notice the purchase attempt and manually intervene if necessary. Thanks again - glad you got yours! I'm also not looking to scalp, just a regular dude looking to outbot the scalpers so I can get a new card.

**Comment by cucco-io** [comment]
@JayV30 did you get yours?

**Comment by JayV30** [comment]
> @JayV30 did you get yours?

I did eventually, using alerts from various discords that are using stock tracking automations. I realized that with the companies now making efforts to discourage automated buying, it would be too much time and effort on my part to maintain my own bot (or use another unless it is VERY actively maintained). Plus, I don't want to get my IP banned from these retailers.

This bot here would no longer work with the changes that best buy has made to their site to discourage bots.

I don't mind using referral links from the discord alerts because those people are actually putting in effort to maintain something (whether they are actually the bot author or not).

@MoonpieQ if you are looking for advice, all I can say is these are the discord servers I follow for this: 

- [Falcodrin Community Discord](https://discord.gg/AcmJ9VSD)
- [CtrlAltStock](https://discord.gg/M9hvjfaz)
- [StockAlerts_DataLover](https://discord.gg/fqrzBmEx)

I'm sure there are many more. If you are looking at running your own bot, [streetmerchant](https://github.com/jef/streetmerchant) seems to be a popular choice.
